pkg/apis/policy/types.go

```diff
@@ -200,23 +200,23 @@ type PodSecurityPolicySpec struct {
 	// privilege escalation. If unspecified, defaults to true.
 	// +optional
 	AllowPrivilegeEscalation bool
-	// AllowedHostPaths is a white list of allowed host paths. Empty indicates that all host paths may be used.
+	// AllowedHostPaths is an allowed list of allowed host paths. Empty indicates that all host paths may be used.
 	// +optional
 	AllowedHostPaths []AllowedHostPath
-	// AllowedFlexVolumes is a whitelist of allowed Flexvolumes.  Empty or nil indicates that all
+	// AllowedFlexVolumes is an allowlist of allowed Flexvolumes.  Empty or nil indicates that all
```

Comment:
I have to run, but it looks like these will need to be checked individually and not just find/replaced.